cmd/plugin/rpaasv2/cmd/info_test.go

```diff
@@ -0,0 +1,147 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli"
+	"gotest.tools/assert"
+)
+
+func setupInfoApp() (*cli.App, *bytes.Buffer) {
+	testApp := NewApp()
+
+	buffer := bytes.NewBuffer(nil)
+	writer := io.Writer(buffer)
+	testApp.Writer = writer
+	testApp.Commands = append(testApp.Commands, Info())
```

Comment:
I think you don't need do that since you're calling the `appendCmds` in `NewApp` function.